src/sentry/templates/sentry/emails/idp_verification_email.txt

```diff
@@ -0,0 +1,5 @@
+Please confirm your email ({{ confirm_email }}) by submitting the following code on the verification page: ({{ verification_key }}).
+
+This code will expire in 10 minutes.
+
+If you did not make this request, you may simply ignore this email.
```

Comment:
want to remove this line? even though its just used internally, if a user did not make this request they shouldn't ignore it.